tools/scanner/src/bin/scan.rs

```diff
@@ -14,6 +14,7 @@
 //! together with the name of the analysis.
 //!
 //! Look at each analysis documentation to see which files an analysis produces.
+#![feature(rustc_private)]
```

Comment:
Why do we need this?

Reply:
Good question! this [commit](736a249954aedaff9b3562f073e295d6c4c6d94d) essentially mandates this change being required

 Without this change, 

we now get the following error:

```
error: cannot satisfy dependencies so `rustc_driver` only shows up once
  |
  = help: having upstream crates all available in one format will likely make this go away
  = note: `rustc_driver` was unavailable as a static crate, preventing fully static linking
  = help: `feature(rustc_private)` is needed to link to the compiler's `rustc_driver` library

error: cannot satisfy dependencies so `core` only shows up once
  |
  = help: having upstream crates all available in one format will likely make this go away
  = help: `feature(rustc_private)` is needed to link to the compiler's `rustc_driver` library
```

which shows that rustc_driver cannot be linked without adding this feature.

Reply:
Link is broken but sounds good :)

Reply:
Fixed link, thanks